tests/test_integration_service.py

```diff
@@ -1115,3 +1116,211 @@ def test_detail_list(self):
                     'role': False,
                 },
             )
+
+
+class TestAddressLookup(util.LoRATestCase):
```

Comment:
Minor nit: this test no longer accesses LoRA directly, so it isn't an integration test, anymore, right? As such, you can make the test suite run ever so slightly faster by descending from `util.TestCase` instead, and preferably also moving it to another file — for consistency.